src/utils/format.ts

```diff
@@ -12,10 +13,15 @@ function _getLocaleSymbols(): { thousands: string; decimals: string } {
     decimals: formattedNumber[5],
   }
 }
+
 const { thousands: THOUSANDS_SYMBOL, decimals: DECIMALS_SYMBOL } = _getLocaleSymbols()
-function _formatNumber(num: string): string {
-  return num.replace(/(\d)(?=(?:\d{3})+(?!\d))/g, '$1' + THOUSANDS_SYMBOL)
+const DEFAULT_THOUSANDS_SYMBOL = ','
+const DEFAULT_DECIMALS_SYMBOL = '.'
+
+function _formatNumber(num: string, thousandsSymbol: string = THOUSANDS_SYMBOL): string {
+  return num.replace(/(\d)(?=(?:\d{3})+(?!\d))/g, '$1' + thousandsSymbol)
```

Comment:
Love this regexp :+1:

Reply:
It's probably yours, I didn't touch it =P